action.yml

```diff
@@ -28,8 +28,10 @@ runs:
         echo "${{ inputs.token }}" | docker login -u ${{ github.actor }} --password-stdin ghcr.io
         docker pull ghcr.io/hashicorp/signore:${{ inputs.version }}
         CONTAINER_ID=$(docker create ghcr.io/hashicorp/signore:${{ inputs.version }})
-        docker cp $CONTAINER_ID:/usr/bin/signore /usr/local/bin/signore
+        mkdir -p $HOME/bin
```

Comment:
I'm sure it's not an issue given how the runners are currently being built, but quoting the use of `$HOME` would make it safer in the case of a space ever showing up in the path.